cpp/ql/test/query-tests/Likely Bugs/Format/NonConstantFormat/nested.cpp

```diff
@@ -78,3 +78,13 @@ void diagnostic(const char *fmt, ...)
 void bar(void) {
     diagnostic (some_instance->get_fmt());  // GOOD
 }
+
+namespace ns {
+
+  class blab {
+    void out1(void) {
+      char *fmt = (char *)__builtin_alloca(10);
+      diagnostic(fmt);  // GOOD
```

Comment:
Surely this is `BAD`, not `GOOD`? It's flagged by the query, so if you think it's `GOOD`, then it's at least `GOOD [FALSE POSITIVE]`. But I think it's `BAD` since the only reason to allocate a writeable buffer is to write non-constant data to it.

Reply:
I think you meant `BAD [FALSE POSITIVE]` in the comment above.  But you're right, this is just plain `BAD`.